tests/conftest.py

```diff
@@ -65,3 +74,60 @@ def image(dev_image):
 def store_path():
     """Path to store zarr images written from benchmarks"""
     return pathlib.Path("data/output/heart-example.zarr")
+
+
+def expand_min_max(config: dict) -> dict:
+    """Expand min/max keys in config to a list of the full range of values."""
+
+    for key in ["chunk_size", "blosc_clevel", "gzip_level", "zstd_level"]:
+        values = config[key]
+        if "min" in values and "max" in values:
+            config[key] = range(values["min"], values["max"] + 1)
+
+    return config
+
+
+def parse_config_files(config_name: str) -> list[dict]:
+    """Parse json config files. 'all' will parse every config in tests/benchmark_configs (except for dev, which contains
+    a small set of parameters for development runs)."""
+
+    configs_dir = pathlib.Path("tests/benchmarks/benchmark_configs")
+    configs = []
+
+    if config_name == "all":
+        for config_file in configs_dir.iterdir():
+            if config_file.is_file() and config_file.stem != "dev":
+                config = expand_min_max(read_json_file(config_file))
+                configs.append(config)
+    else:
```

Comment:
Should there also be a branch for 'dev' here? Or am I misunderstanding?

Reply:
Thanks for the input @dstansby ! 

I don't think it's necessary to have another branch - `dev` is a config file like any of the others in `tests/benchmarks/benchmark_configs`, it just contains a small number of options for each parameter so it runs quickly (it's also the default option if no `config` is provided).

So running e.g. the `shuffle` config:
```
tox -- --dev-image --config=shuffle
```

Works the same as running the `dev` config:
```
tox -- --dev-image --config=dev
```

I've excluded `dev` from `--all`, as it doesn't contain parameter combinations we'll actually use in plots.

Reply:
That makes sense 👍